satellite/metainfo/validation.go

```diff
@@ -366,26 +366,34 @@ func (endpoint *Endpoint) validatePieceHash(ctx context.Context, piece *pb.Remot
 	defer mon.Task()(&ctx)(&err)
 
 	if piece.Hash == nil {
-		return errs.New("no piece hash, removing from pointer %v (%v)", piece.NodeId, piece.PieceNum)
+		return errs.New("no piece hash. NodeID: %v, PieceNum: %d)", piece.NodeId, piece.PieceNum)
```

Comment:
The ending parenthesis in `PieceNum: %d)` looks odd. Perhaps, a missing one before `Node ID: %v`?

Reply:
I missed deleting it when I refactored the message.